acceptance/testdata/detector/container/cnb/buildpacks/simple_buildpack/simple_buildpack_version/bin/detect

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+plan_path=$2
+
+cat >> "${plan_path}" <<EOL
+[[provides]]
+name = "some_requirement"
+
+[[requires]]
+name = "some_requirement"
+version = "0.1"     # Optional
```

Comment:
Given this buildpack uses API `0.3`, we should move `requires.version` into `requires.metadata.version` unless we are explicitly testing the warning message for the top-level version.